test/constant_test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import {constant} from '../src/constant.js';
+import * as utils from './utils.js';
+
+
+describe('test constant', function() {
+  function testConstant(start, step, outputShape, type, expected) {
+    const outputTensor = constant(start, step, outputShape, type);
```

Comment:
There *may* be some fallout from the discussion of https://github.com/webmachinelearning/webnn/issues/492, but it would be a minor change. e.g.

`const outputTensor = constant({dimensions: outputShape, dataType: dataType}, start, step);`